cmd/openshift-tuned.go

```diff
@@ -815,6 +841,13 @@ func changeWatcher() (err error) {
 			if err := pullLabels(clientset, &tuned, nodeName); err != nil {
 				return err
 			}
+			if resyncPeriod.now/2 >= resyncPeriod.min {
+				// we've increased the original resyncPeriod due to errors; there was a successful
+				// pull now, converge back to the original resyncPeriod.min value
+				resyncPeriod.now /= 2
```

Comment:
This is to slowly bring the resyncPeriod back to original?

Reply:
Yes.